docker_release/main.py

```diff
@@ -207,9 +207,9 @@ def main():
             image = '%s/%s:latest' % (organization, repository)
             _docker_build(docker, args, docker_image_dir, image)
 
-            latest_tag = 'latest-snapshot'
```

Comment:
Yeah, but somebody else could push a snapshot right after you and latest-snapshot doesn't point to what you think it does. It's only useful if only one person is iterating on the docker images at a time.